app/middlewares/errors.js

```diff
@@ -9,7 +9,8 @@ const statusCodes = {
   [errors.API_ERROR]: 500,
   [errors.BAD_REQUEST_ERROR]: 400,
   [errors.NOT_FOUND_ERROR]: 404,
-  [errors.CONFLICT_ERROR]: 409
+  [errors.CONFLICT_ERROR]: 409,
+  [errors.SCHEMA_ERROR]: 400
```

Comment:
check the `422` error code, it's more appropriate for this type of error than a `400`